cfgov/unprocessed/css/cf-enhancements.less

```diff
@@ -520,6 +520,29 @@ tbody {
 
 
 /* topdoc
+  name: Hide
+  family: cf-core
+  patterns:
+    - name: Utility class
+      markup: |
+        <h1 class="u-hide">
+          Show using JavaScript
+        </h1>
+      codenotes:
+        - .u-hide;
+      notes:
+        - "Use this class to hide something,
+          perhaps to be shown using JavaScript."
+  tags:
+    - cf-core
+*/
+
+.u-hide {
```

Comment:
Yep. Down in cf-enhancements, there's that an a class to only hide when JS is enabled.